x-pack/plugins/maps/public/embeddable/map_component.tsx

```diff
@@ -25,6 +26,7 @@ export interface Props {
   isLayerTOCOpen?: boolean;
   mapCenter?: MapCenterAndZoom;
   onInitialRenderComplete?: () => void;
+  renderTooltipContent?: RenderToolTipContent;
```

Comment:
Lets call this `getTooltipRenderer`. Its also added in https://github.com/elastic/kibana/pull/182737. The reason for `get` is that its value is only used once and not updated when the value changes. If its just `renderTooltipContent` then users of the Component would expect that changing the prop would be reflected in the component.